$NetBSD: patch-aa,v 1.7 2008/12/19 21:09:11 heinz Exp $

  - At the moment, this is only for NetBSD

  - Override *OWN and *GRP variables for DESTDIR support.

--- Makefile.orig	2003-08-28 16:43:35.000000000 +0200
+++ Makefile
@@ -38,8 +38,8 @@
 
 # openbsd means OpenBSD 2.5 or newer. freeswan is the name for Linux with
 # FreeS/WAN integrated, freebsd/netbsd means FreeBSD/NetBSD with KAME IPsec.
-OS=		openbsd
-#OS=		netbsd
+#OS=		openbsd
+OS=		netbsd
 #OS=		freebsd
 #OS=		freeswan
 #OS=		bsdi
@@ -55,6 +55,12 @@ FEATURES+=	rawkey isakmp_cfg
 .PATH:		${.CURDIR}/sysdep/${OS}
 
 PROG=		isakmpd
+
+BINOWN=		@BINOWN@
+BINGRP=		@BINGRP@
+MANOWN=		@MANOWN@
+MANGRP=		@MANGRP@
+
 BINDIR?=	/sbin
 LDSTATIC?=	-static
 SRCS=		app.c attribute.c cert.c connection.c constants.c conf.c \
@@ -71,7 +77,7 @@ CLEANFILES=	exchange_num.c exchange_num.
 		isakmp_num.c isakmp_num.h ipsec_fld.c ipsec_fld.h \
 		isakmp_fld.c isakmp_fld.h
 MAN=		isakmpd.8 isakmpd.conf.5 isakmpd.policy.5
-CFLAGS+=	-Wall -Wstrict-prototypes -Wmissing-prototypes \
+CFLAGS+=	-Wall -Wmissing-prototypes \
 		-DNEED_SYSDEP_APP \
 		-I${.CURDIR} -I${.CURDIR}/sysdep/${OS} -I.
 #CFLAGS+=	-Wsign-compare -Werror
